Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2832,6 +2838,10 @@ namespace OpenInfraPlatform
 					// if the radial vector from Center to a point has coordinates (a,b), then the direction vector at that point is (−b,a) 
 					double xRadius = ellipse->SemiAxis1 * this->UnitConvert()->getLengthInMeterFactor();
 					double yRadius = ellipse->SemiAxis2 * this->UnitConvert()->getLengthInMeterFactor();
+					double xPoint = xRadius * cos(angle);
+					double yPoint = yRadius * sin(angle);
+
+
 					carve::geom::vector<3> direction = carve::geom::VECTOR(yRadius * -sin(angle), xRadius * cos(angle), 0.);
 					return direction.normalize();

```

Comment:
You declare these variables but don't use them.

Reply:
I am still not sure about solution to this, thus I added radiuses to try some calculations

Reply:
Resolved: f743c7c